<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Preeti" name="testCases.myPIM.MTA_9_ValidLogin" tests="2" failures="0" timestamp="14 Feb 2018 07:16:05 GMT" time="41.637" errors="1">
  <testcase name="closeBrowser" time="0.000" classname="testCases.myPIM.MTA_9_ValidLogin">
    <error type="org.testng.TestNGException" message="
Can inject only one of &lt;ITestContext, XmlTest&gt; into a @AfterClass annotated closeBrowser.
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: Can inject only one of <ITestContext, XmlTest> into a @AfterClass annotated closeBrowser.For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injectionat org.testng.internal.Parameters.checkParameterTypes(Parameters.java:407)at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:356)at org.testng.internal.Parameters.createParameters(Parameters.java:635)at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:190)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:209)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:208)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- closeBrowser -->
  <testcase name="ValidUserLogin" time="41.637" classname="testCases.myPIM.MTA_9_ValidLogin"/>
</testsuite> <!-- testCases.myPIM.MTA_9_ValidLogin -->
